development/extensions/new_in_rhea.rst

```diff
@@ -98,15 +98,13 @@ Fortunately, the integration is pretty seamless and most existing extensions tha
   * ``generate_text_for_edit()``
   * ``generate_text_for_storage()``
   * ``strip_bbcode()``
-  * ``smiley_text()``
```

Comment:
Why did you remove smiley_text()? It's still available in 3.2 and can still be used as it behaved in 3.1.